mysql-plugin/src/e2e-test/features/mysqlsink/DesignTimeValidation.feature

```diff
@@ -0,0 +1,179 @@
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License
+
+Feature: MySQL Sink - Design time validation scenarios
+
+  @TS-MYSQL-SINK-DSGN-ERROR-01
+  Scenario: Verify Database field validation error message with invalid test data
```

Comment:
Source node is not required for this scenario, please remove the source node details from this scenario.

Reply:
So, here to get errors for invalid fields , we need to configure the source plugin also, otherwise we will not get error message for the invalid values entered in the sink plugin